pkg/cicd/circleci/circleci.go

```diff
@@ -81,3 +81,23 @@ func (p *Provider) GetTemplateFile() string {
 
 	return filepath.Join(p.GetWorkDirectory(), ".krew.yaml")
 }
+
+// GetKrewIndexRepoName gets upstream_krew_index_repo_name
+func (p *Provider) GetKrewIndexRepoName() string {
+	nameInput := getInputForAction("UPSTREAM_KREW_INDEX_REPO_NAME")
```

Comment:
to be consistent with other places, we should keep it same as the input as defined in action.yml, so kindly change it to upstream_krew_index_repo_name

Reply:
no problem. I will update the fields to be lowercase.